yts/yts.py

```diff
@@ -13,13 +13,29 @@
 class YTS:
     """ provides an API for downloading yts yifi movies. """
 
-    # Initialize with the current yts url
     def __init__(self, yts_url):
+        """
+        Initialize with the current yts url 
+
+        Parameters
+        ----------
+            yts_url: str, required
+                The URL of the yts website
+        """
+
         self.url = yts_url
 
-    # Gets the popular downloads from the homepage
-    # Returns a dictionary with movie title and rating
     def get_popular_downloads(self) -> dict:
+        """
+        Gets the popular downloads from the homepage
+        Returns a dictionary with movie title and rating 
+
+        Returns
+        -------
+            pop_movies_dict: dictionary, { title: str, rating: str }
```

Comment:
This ```{ title: str, rating: str }``` is inconsistent with [Line #63](https://github.com/darkHarry/yts-cli/pull/7/files#diff-8711f00eeac89bdaf2440f5776745f59aa32f26c3e80ca012a903a4e781ef801R63).
Can you change this to how you described params with its data type in Line #63.